### ISSUE TITLE
Staging failing

### ISSUE THREAD

**Issue by diversemix** [elifesciences/elife-xpub#822]
See attached image.

![image](https://user-images.githubusercontent.com/8524685/47157220-c6865600-d2e0-11e8-8736-eb0427b093f4.png)

**Comment by tamlyn** [comment]
Yes, poor error handling on my part. Will fix. The underlying cause on staging is that the GraphQL request failed because the user was not found in the profiles API.

**Comment by diversemix** [comment]
ok - I fixed things my side by clearing the site data.

**Comment by diversemix** [comment]
Closing as this was an issue with my local storage ... but did generate changes by @tamlyn  to the AppBar error handing see #828
